capstone.html

```diff
@@ -0,0 +1,1062 @@
+						</li>
+						<li>
+							Tighten the screws until the hex nuts “catch” them and the ball caster is tightly bound.
+						</li>
+					</ul>
+					<img class="pj-img-fluid mx-auto mb-0" src=".\assets\images\capstone\ops_capstone_chassis_grip_parts.png" 
+					width="550">
+					<div style="text-align: center;">
+						<img class="pj-img-fluid mx-auto mb-0" src=".\assets\images\capstone\ops_capstone_chassis_grip_1.png" 
+						width="150" style="display: inline;">
+						<img class="pj-img-fluid mx-auto mb-5" src=".\assets\images\capstone\ops_capstone_chassis_grip_2.png" 
+						width="175" style="display: inline;">
+					</div>
+				</li>
+				<li class="list-group-item">
+					Clip the chassis grip to the edge of the plate opposite to the battery holder. Center align the chassis grip 
+					along the edge as well.
+					<img class="pj-img-fluid mx-auto mb-5" src=".\assets\images\capstone\ops_capstone_chassis_grip_attached.png" 
+					width="600">
+				</li>
+				<li class="list-group-item">
+					Attach the wheels to the gearbox motors, the metal ball to the caster, and the 9V battery to the battery holder. 
+					Note that the battery slides into the holder as shown below.<br>
+					<img class="pj-img-fluid mx-auto mb-5" src=".\assets\images\capstone\ops_capstone_battery_holder_and_rover.png" 
+					width="800">
+				</li>
+				<li class="list-group-item">
+					Attach the wheels to the gearbox motors, the metal ball to the caster, and the 9V battery to the battery holder. 
+					Note that the battery slides into the holder as shown below.<br>
+					<img class="pj-img-fluid mx-auto mb-5" src=".\assets\images\capstone\ops_capstone_breadboard_and_rover.png" 
+					width="800">
+				</li>
+				<li class="list-group-item">
+					The final step is to place and wire the electrical components on the breadboard. Remember that there is no 
+					schematic. You have to design the circuit yourself.
+					<br><br>
+					Try drawing up your own schematic on <a class="link-primary" href="https://www.tinkercad.com/circuits">Tinkercad</a> 
+					first. You can use all the same components and even simulate your circuit with code!
+					<br><br>
+					If you get stuck, <b>ask questions</b>.
+					<img class="pj-img-fluid mx-auto mb-0" src=".\assets\images\capstone\ops_capstone_rover_done.png" 
+					width="800">
+				</li>
+				<br>
+			</ol>
+			
+			<h4 class="mb-3 fw-light">Programming</h4>
+			<p class="mb-5">
+				How you write the code is all up to you. Just make sure there are two files, one for the rover and one for the remote control.
+				<br><br>
+				<b>Install the newest version</b> of the NRF24L01_W library as well.
+				<br><br>
+				As a final suggestion, look back at your old project submissions and reuse applicable code.
+			</p>
+
+			<h2 class="mb-3 fw-bold" id="pj-heading-references">References</h2>
+			<ul>
+				<li>
+					<a class="link-primary" 
```

Comment:
imo for all of these links, I would open them in a new tab

This is done by adding:

`target="_blank" rel="noopener noreferrer"` to the anchor tag.

If you're confused, google "html open anchor in new tab"

Reply:
Should these pdfs be downloaded on click like the others or still just be site links in another tab?

Reply:
pdfs should open up to pdfs in a new tab also